scripts/check-workspace.sh

```diff
@@ -17,19 +17,17 @@
 
 declare -A results
 
-all_crates=("core" "alloc" "utils" "model" "lang" "lang" "lang/macro" "cli")
-wasm_crates=("core" "alloc" "utils" "model" "lang" "lang" "lang/macro")
+all_crates=("env" "storage" "storage/derive" "allocator" "prelude" "primitives" "lang" "lang/macro" "lang/ir")
```

Comment:
`lang/codegen` missing

Reply:
`metadata` missing

Reply:
So I copied this from out [.gitlab-ci.yml](https://github.com/paritytech/ink/blob/master/.gitlab-ci.yml#L20). Are they missing there intentionally or should we add them there as well?